styles/index.css

```diff
@@ -43,12 +43,13 @@ body {
   border-radius: 20px;
   flex-direction: column;
   padding: 20px;
+  position: relative;
 }
 
 .main-card__bookmark {
-  position: relative;
-  right: -170px;
-  top: -25px;
+  position: absolute;
+  right: 20px;
+  top: -5px;
 }
 
 .Que {
```

Comment:
I was confused by this abbreviation, maybe "question" would be a better option?

Reply:
Absolutely!